app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -479,10 +486,10 @@ data class PocketStoriesCategoryColors(
          */
         @Composable
         fun buildColors(
-            selectedBackgroundColor: Color = FirefoxTheme.colors.actionPrimary,
-            unselectedBackgroundColor: Color = FirefoxTheme.colors.actionTertiary,
-            selectedTextColor: Color = FirefoxTheme.colors.textActionPrimary,
-            unselectedTextColor: Color = FirefoxTheme.colors.textActionTertiary,
+            selectedBackgroundColor: Color = SelectableChipColors.buildColors().selectedBackgroundColor,
+            unselectedBackgroundColor: Color = SelectableChipColors.buildColors().unselectedBackgroundColor,
+            selectedTextColor: Color = SelectableChipColors.buildColors().selectedTextColor,
+            unselectedTextColor: Color = SelectableChipColors.buildColors().unselectedTextColor,
```

Comment:
Do we need `PocketStoriesCategoryColors` since this is really just a Selectable Chip now?

Reply:
I'll remove `PocketStoriesCategoryColors`